docs/implementation-guides/syncs/implement-a-sync.mdx

```diff
@@ -183,12 +183,20 @@ Easily test your integration functions locally as you develop them with the `dry
 nango dryrun salesforce-contacts '<CONNECTION-ID>'
 ```
 
-To learn more about all the options for dryrun, run: `nango dryrun --help`
+You can also enable performance diagnostics to monitor memory usage and CPU metrics:
+
+```bash
+nango dryrun salesforce-contacts '<CONNECTION-ID>' --diagnostics
+```
+
+The `--diagnostics` flag displays detailed performance metrics including average and peak memory usage (RSS, heap, external) and CPU utilization, which is useful for performance tuning and identifying memory leaks.
+
+To learn more about all the options for the dryrun, run: `nango dryrun --help`.
 
```

Comment:
Can we add a disclaimer stating that local diagnostics are only indicative and do not fully represent the cpu/memory consumption when run by Nango cloud. 
I would prefer not to see customer support tickets stating that "it works perfectly fine on their machine"

@bastienbeurier @khaliqgant